lib/Carton/Doc/FAQ.pod

```diff
@@ -109,4 +109,12 @@ L<Any::Moose>.
 
 =back
 
+=head2 I'm using perlbrew, but C<< carton install >> uses system perl, how to fix?
 
+Instead of C<< carton install >>, use C<< perl -S carton install >>.
+
+This will force the use of the C<perl> from your PATH, and not the C</usr/bin/perl>
+that is hard-coded at the top of the C<carton> script.
```

Comment:
It is _not_ hardcoded, it will be fixed to your whatever perl path upon installation by MakeMaker.